src/templates/ja/2020/methodology.html

```diff
@@ -360,39 +365,87 @@ <h3 id="rework-css"><a href="#rework-css" class="anchor-link">Rework CSS</a></h3
       </p>
     </section>
 
+    <section>
+      <h4 id="rework-utils"><a href="#rework-utils" class="anchor-link">Rework Utils</a></h4>
+
+      <p>
+        今年の<a href="{{ url_for('chapter', year=year, lang=lang, chapter='css') }}">CSS</a>の章では、<a href="{{ url_for('contributors', year=year, lang=lang, _anchor='LeaVerou') }}">Lea Verou</a>がCSSの状態をかなり詳細に見ており、<a href="https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/sql/2020/01_CSS">100以上のクエリ</a>にわたっています。視点を変えれば、2019年の2.5倍のクエリ数になります。この規模の分析を実現可能にするために、Leaは<a href="https://github.com/LeaVerou/rework-utils">Rework Utils</a>をオープンソース化しました。これらのユーティリティは、CSSの洞察をより簡単に抽出するための有用なスクリプトを提供することで、ReworkからJSONデータを次のレベルに引き上げます。CSSの章で見る統計情報のほとんどは、これらのスクリプトによって提供されています。
+      </p>
+    </section>
+
+    <section>
+        <h4 id="parsel"><a href="#parsel" class="anchor-link">Parsel</a></h4>
+
+        <p>
+            <a href="https://projects.verou.me/parsel/">Parsel</a>はCSSセレクタパーサとspecificity calculatorで、元々は<a href="{{ url_for('chapter', year=year, lang=lang, chapter='css') }}">CSS</a>の章をリードする<a href="{{ url_for('contributors', year=year, lang=lang, _anchor='LeaVerou') }}">Lea Verou</a>と別のライブラリとしてオープンソース化されています。
+            これは、セレクタと特異性に関連するすべてのCSSメトリクスで広く使われています。
+        </p>
+    </section>
+
     <section>
       <h2 id="process"><a href="#process" class="anchor-link">分析プロセス</a></h2>
 
       <p>
-        Web Almanacの計画と実行には、Webコミュニティの数十人の貢献者の協力を得て、約1年を要しました。このセクションでは、Web Almanacに掲載されているメトリクスを選択した理由、それらがどのようにして照会されたか、そして解釈されたかについて説明します。
+        Web Almanacの計画と実行には、100人以上のWebコミュニティの<a href="{{ url_for('contributors', year=year, lang=lang) }}">投稿者</a>の協力を得て、約1年を要しました。このセクションでは、Web Almanacに掲載されているチャプターを選んだ理由、それらのメトリクスがどのようにして照会されたか、そしてどのように解釈されたかを説明します。
       </p>
     </section>
 
     <section>
-      <h3 id="brainstorming"><a href="#brainstorming" class="anchor-link">ブレインストーミング</a></h3>
+      <h3 id="planning"><a href="#planning" class="anchor-link">プランニング</a></h3>
+
+      <p>
+        2020年のWeb Almanacは、<a href="https://twitter.com/rick_viscomi/status/1273135952848977920">2020年6月</a>にキックオフしましたが、COVID-19に関連した不安や社会正義の抗議活動のため、2019年のタイムラインよりも遅くなりました。2020年のこれらの出来事やその他の出来事は、制作プロセス全体の下火となり、このようなテンポの速いプロジェクトのストレスを超えて、私たちの貢献者に多くの追加的な負担をかけました。
```

Comment:
@ksakae1216 
kicked off = 開始、発端
https://eow.alc.co.jp/search?q=kick+off
```suggestion
        2020年のWeb Almanacは、<a href="https://twitter.com/rick_viscomi/status/1273135952848977920">2020年6月</a>に開始されましたが、COVID-19に関連した不安や社会正義の抗議活動のため、2019年のタイムラインよりも遅くなりました。2020年のこれらの出来事やその他の出来事は、制作プロセス全体の下火となり、このようなテンポの速いプロジェクトのストレスを超えて、私たちの貢献者に多くの追加的な負担をかけました。
```

Reply:
I fix.